### ISSUE TITLE
[BUG] Previews not working on firefox

### ISSUE THREAD

**Issue by tefkah** [hauptrolle/chakra-templates#139]
Thanks for the fantastic website and templates!

However, the previews don't seem to work for me in Firefox.
## Page

In Firefox:
<img width="1360" alt="image" src="https://user-images.githubusercontent.com/21983833/142218335-aca2223a-15eb-4a3c-abb9-e300f89897ab.png">

In Brave:

<img width="1360" alt="image" src="https://user-images.githubusercontent.com/21983833/142218652-ae122138-872f-487e-b7e8-35e7b1c245dd.png">

## Network:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/21983833/142218147-d7dd2e8f-2248-441e-859c-7079bf981eda.png">

## Info

No specific tracker blocking or anything that would normally break these kinds of things.

MacBook Air M1, MacOS Monterey, Firefox 94.0.1

**Comment by hauptrolle** [comment]
This is probably the same issue as #128. Will have a look at it, and maybe finally refactor the preview frame to use https://github.com/codesandbox/sandpack :)

**Comment by populationtwo** [comment]
This appears to be caused by [Enhanced Tracking Protection](https://support.mozilla.org/en-US/kb/enhanced-tracking-protection-firefox-desktop) on Firefox. The preview frame works fine if ETP is disabled.

@hauptrolle  I can look into refactoring to Sandpack if you'd like to go that route.

**Comment by hauptrolle** [comment]
Hey @populationtwo I have the problem now in Chrome too...
But yes, I think we should go the way, and try out sandpack. We can have a chat about that on the discord 👍

**Comment by rvalle** [comment]
I am locked out of chakra templates... not working on any of my browsers. Also getting grey boxes.

**Comment by NSDrowned** [comment]
Same here... gray boxes in Chrome

**Comment by dbruvers** [comment]
The previews do work if you click the "Open in Fullscreen" button
![open in fullscreen](https://user-images.githubusercontent.com/15773882/159939941-630b3b81-9c23-4450-981a-eacf77c6f45f.png)

**Comment by henrychea** [comment]
Can concur, not working in Chrome as well.

macOS 12.3.1 , Chrome Version 100.0.4896.75 (Official Build) (x86_64)

<img width="1552" alt="image" src="https://user-images.githubusercontent.com/40254517/162177608-52dc055e-a8df-4c0d-b948-d1cc683ed928.png">

**Comment by statico** [comment]
I tried running this project locally and can't get them to show up in Firefox or Chrome either.

**Comment by henrychea** [comment]
Update with some info from Firefox dev tools `The resource from “https://fonts.googleapis.com/css?family=” was blocked due to MIME type (“text/html”) mismatch (X-Content-Type-Options: nosniff).` Maybe this can help.

**Comment by henrychea** [comment]
This seems to be an issue with the Google Font's Api. Maybe someone could submit a pull request to update the font link and all the bugs should go away.

```Refused to apply style from '<URL>' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
Refused to apply style from '<URL>' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
Refused to apply style from '<URL>' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
Refused to apply style from '<URL>' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
Refused to apply style from '<URL>' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
Refused to apply style from '<URL>' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
Refused to apply style from '<URL>' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
chakra-templates.dev/:1 Refused to apply style from 'https://fonts.googleapis.com/css?family=' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
chakra-templates.dev/:1 Refused to apply style from 'https://fonts.googleapis.com/css?family=' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
callToActionWithIllustration:1 Refused to apply style from 'https://fonts.googleapis.com/css?family=' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
splitScreen:1 Refused to apply style from 'https://fonts.googleapis.com/css?family=' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
callToActionWithVideo:1 Refused to apply style from 'https://fonts.googleapis.com/css?family=' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
callToActionWithAnnotation:1 Refused to apply style from 'https://fonts.googleapis.com/css?family=' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
withBackgroundImage:1 Refused to apply style from 'https://fonts.googleapis.com/css?family=' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
```

**Comment by henrychea** [comment]
I have an update for all Firefox users, you can get the preview to work by clicking any link that goes to an external site (use the contribute link in the navbar). Then just navigate back and the preview works.

**Comment by hauptrolle** [comment]
closed in favor if #198
